clients/admin-ui/src/features/user-management/UserManagementRow.tsx

```diff
@@ -87,6 +93,23 @@ const UserManagementRow: React.FC<UserManagementRowProps> = ({ user }) => {
             </Badge>
           ))}
         </Td>
+        <Td pl={0} py={1} onClick={handleEditUser}>
```

Comment:
I realize this wasn't introduced here, but why isn't the `onClick={handleEditUser}` on the `<Tr>` instead of every `<Td>`?

Reply:
It's on every `<Td>` because the `<Tr>` encompasses the kebab `...` menu to edit/delete user. There might be a better way to exclude that specific menu item from that click event, but this was one way around it.